.github/workflows/check.yml

```diff
@@ -95,19 +95,20 @@ jobs:
           submodules: true
       - uses: dtolnay/rust-toolchain@nightly
         with:
-          components: rustc, rust-std, cargo, llvm-tools
+          components: rustc, rust-std, cargo, llvm-tools, llvm-tools-preview
       - uses: baptiste0928/cargo-install@v3
         with:
           crate: grcov
       - run: cargo clean
       - run: cargo test --all-features --no-fail-fast
         env:
           CARGO_INCREMENTAL: '0'
-          RUSTFLAGS: '-Zprofile -Ccodegen-units=1 -Cllvm-args=--inline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Cpanic=abort -Zpanic_abort_tests'
+          LLVM_PROFILE_FILE: 'target/debug/coverage/dcaf-%p-%m.profraw'
+          RUSTFLAGS: '-Cinstrument-coverage -Cpanic=abort -Zpanic_abort_tests'
           RUSTDOCFLAGS: '-Zprofile -Ccodegen-units=1 -Cllvm-args=--inline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Cpanic=abort -Zpanic_abort_tests'
       - run: mkdir ./target/debug/coverage
-      - run: zip ./target/debug/coverage/files.zip ./target/debug/deps/dcaf-*.gcda ./target/debug/deps/dcaf-*.gcno
-      - run: grcov ./target/debug/coverage/files.zip -s . --service-name "Continuous Integration" --commit-sha ${GITHUB_SHA} -t coveralls --branch --ignore-not-existing --ignore "../*" --ignore "/*" --ignore "*/test_helper.rs" -o ./target/debug/coverage/ --token=${COVERALLS_TOKEN}
+      - run: zip ./target/debug/coverage/files.zip ./target/debug/deps/dcaf-*.profraw
```

Comment:
That part wasn't changed by this commit, but should we put this in quotes in case the `*` contains spaces?

Reply:
Correction: Putting this in quotes will cause the zip-Command to fail, so we must not use quotes here.

As the filenames follow the scheme in `LLVM_PROFILE_FILE` (see also [the Rust compiler documentation regarding the naming scheme](https://doc.rust-lang.org/nightly/rustc/instrument-coverage.html#running-the-instrumented-binary-to-generate-raw-coverage-profiling-data)), there wouldn't be any files with spaces in there anyways, so this should be fine.